main.go

```diff
@@ -114,12 +116,30 @@ func main() {
 	eventRouter := NewEventRouter(clientset, eventsInformer)
 	stop := sigHandler()
 
-	// Startup the http listener for Prometheus Metrics endpoint.
+	mux := http.NewServeMux()
```

Comment:
We need to create our own mux here so we can *optionally* add the pprof handler, with the `DefaultMux` importing `net/http/pprof` adds the handler automatically.